code/modules/surgery/autodoc.dm

```diff
@@ -93,7 +93,7 @@
 				patchnote.surgery_operations |= AUTODOC_FRACTURE
 		if(AUTODOC_EMBED_OBJECT in possible_operations)
 			if(external.implants)
-				if(/obj/item/material/shard/shrapnel in external.implants)
+				if(locate(/obj/item/material/shard/shrapnel) in external.implants) //occy edit
```

Comment:
It seems like this was implemented upstream at some point, so the edit comment isn't required, woo!
```suggestion
				if(locate(/obj/item/material/shard/shrapnel) in external.implants)
```